### ISSUE TITLE
SystemVM Template Upgrade doesn't kick in for NoopDbUpgrade

### ISSUE THREAD

**Issue by rohityadavcloud** [apache/cloudstack#6980]
```
2022-12-12 23:27:54,568 DEBUG [c.c.u.d.VersionDaoImpl] (main:null) (logid:) Checking to see if the database is at a version before it was the version table is created
2022-12-12 23:27:54,611 DEBUG [c.c.u.SystemVmTemplateRegistration] (main:null) (logid:) Looking for file [ ./engine/schema/dist/systemvm-templates/metadata.ini ] in the classpath.
2022-12-12 23:27:54,611 DEBUG [c.c.u.SystemVmTemplateRegistration] (main:null) (logid:) Looking for file [ /usr/share/cloudstack-management/templates/systemvm/metadata.ini ] in the classpath.
2022-12-12 23:27:54,624 INFO  [c.c.u.DatabaseUpgradeChecker] (main:null) (logid:) DB version = 4.17.1.0 Code Version = 4.17.2.0
2022-12-12 23:27:54,625 INFO  [c.c.u.DatabaseUpgradeChecker] (main:null) (logid:) Database upgrade must be performed from 4.17.1.0 to 4.17.2.0
2022-12-12 23:27:54,635 DEBUG [c.c.u.DatabaseUpgradeChecker] (main:null) (logid:) Running upgrade NoopDbUpgrade to upgrade from 4.17.1.0-4.17.2.0 to 4.17.2.0
2022-12-12 23:27:54,651 INFO  [c.c.u.DatabaseUpgradeChecker] (main:null) (logid:) Cleanup upgrade NoopDbUpgrade to upgrade from 4.17.1.0-4.17.2.0 to 4.17.2.0
2022-12-12 23:27:54,668 DEBUG [c.c.u.DatabaseUpgradeChecker] (main:null) (logid:) Upgrade completed for version 4.17.2.0
2022-12-12 23:27:54,669 INFO  [o.a.c.s.l.CloudStackExtendedLifeCycle] (main:null) (logid:) Configuring CloudStack Components
2022-12-12 23:27:54,671 INFO  [o.a.c.s.l.CloudStackExtendedLifeCycle] (main:null) (logid:) Done Configuring CloudStack Components
2022-12-12 23:27:54,671 INFO  [o.a.c.s.m.m.i.DefaultModuleDefinitionSet] (main:null) (logid:) Loaded module context [system] in 1108
```

**Comment by rohityadavcloud** [comment]
Fixed with https://github.com/apache/cloudstack/pull/6981

**Comment by DaanHoogland** [comment]
seems related to https://github.com/apache/cloudstack/pull/6858 @rohityadavcloud ?

**Comment by DaanHoogland** [comment]
@rohityadavcloud please review/test with #6858 merged. I am inclined to remove this from the 4.18 milestone as it is not a regression/blocker and does not have a PR linked otherwise.

**Comment by DaanHoogland** [comment]
fixed in #7564
